Tests/conf.json

```diff
@@ -639,7 +639,7 @@
         },
         {
             "integrations": "Generic SQL",
-            "playbookID": "generic-sql",
+            "playbookID": "generic-sql-oracle",
```

Comment:
what will happen to `generic-sql` do you plan to stop running it?

Reply:
it will run for mysql, one version of mssql (there is another test playbook that run with the other one) and postgesql.